ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -36,6 +39,12 @@ const server = (DB) => {
     process.exit(0)
   }
 
+
+  /**
+   * start secret delete thread
+   */
+  init_secret_delete_thread(DB)

```

Comment:
why not follow the line#51 to name it as a xxx_timer()?

Reply:
what about _secret_delete_timer?